settings.gradle

```diff
@@ -2,3 +2,4 @@ rootProject.name='triplea'
 include 'game-core'
 include 'lobby'
 include 'lobby-db'
+include 'test-util'
```

Comment:
Not sure about the `-util` suffix.  How about `-core` or `-common`?

Reply:
`-common` brings back some bad memories ;)    But it works pretty well for this case, this is the common testing project. I'm nervous we may not create new specific test projects for other less generic aspects of testing. Regardless, happy enough to update to `-common`:   32a50ca